arku/parser.py

```diff
@@ -5,9 +5,15 @@
 from contextvars import ContextVar
 from typing import Any, Dict, Generator, Optional, Type, Union
 
-from redis.asyncio.connection import SERVER_CLOSED_CONNECTION_ERROR, Connection, Encoder, HiredisParser, PythonParser
+from redis.asyncio.connection import Connection, Encoder
 from redis.typing import EncodableT, EncodedT
 
+try:
```

Comment:
It is better to separate the imports, one import - one fallback.

Reply:
Fixed